spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -578,6 +578,24 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("like with custom escape") {
+    val table = "names"
+    withTable(table) {
+      sql(s"create table $table(id int, name varchar(20)) using parquet")
+      sql(s"insert into $table values(1,'James Smith')")
+      sql(s"insert into $table values(2,'Michael_Rose')")
+      sql(s"insert into $table values(3,'Robert_R_Williams')")
+
+      // Filter column having values that include underscores
+      val queryDefaultEscape = sql("select id from names where name like '%\\_%'")
+      checkAnswer(queryDefaultEscape, Row(2) :: Row(3) :: Nil)
```

Comment:
Strictly speaking, `checkSparkAnswerAndOperator()` to be used to make sure native comet is actually executed?

Reply:
Good point. @sujithjay could you make that change?

Reply:
Done in https://github.com/apache/datafusion-comet/pull/478/commits/496e885da9c938519822bf021505e7fcf90395b5. Please review @kazuyukitanimura @andygrove.